docs/archival.rst

```diff
@@ -52,7 +52,11 @@ preferred formats.  An example of this is:
 In this case, note that the pysat 'name' label is output to three different
 metadata values required by the ITSP standards. Additionally, the
 :py:attr:`export_pysat_info` option is set to false here. This drops several
-internal :py:mod:`pysat` metadata values before writing to file.
+internal :py:mod:`pysat` metadata values before writing to file. Note that
+this includes the default acknowledgements and references objects. These
+are set manually to avoid conflicts between the original dataset and the
```

Comment:
I checked out the REACH script but I don't see where an acknowledgement or references is set. Want to make sure users understand how to set information in this configuration.

Reply:
should be fixed with a reference to the point in the code